cmd/cli/pigo.go

```diff
@@ -28,7 +28,12 @@ func Run() error {
 	flag.BoolVar(&record, "record", false, "wheter or not to record instead of intercept")
 	flag.Parse()
 
-	interceptConfig, err := config.LoadConfig(configPath)
+	// Load configuration and watch for changes during runtime
+	interceptConfig := &config.Config{}
```

Comment:
this can be a singleTone in the config Package instead! The first time it runs then creates the pointer, the rest just update.